src/content/blog/open-source-pledge.md

```diff
@@ -0,0 +1,51 @@
+
+The Open Source Pledge aligns with DDEV’s goals by setting a clear benchmark for funding. Agencies and other organizations that rely on DDEV as part of their toolchain can use this pledge to estimate their contribution to projects they depend on.
+
+## Price Anchoring: Why $2000 Matters
+
+The USD$2000 per-developer per-year price point serves as a clear anchor, guiding organizations to think practically about supporting open source. Consider a web development agency using DDEV alongside other tools like CMS platforms, CI/CD systems, and libraries. If DDEV represents 10% of their overall toolchain, the agency might allocate $200 per developer per year to support DDEV.
+
+While the percentage will vary, the Open Source Pledge provides a starting point for companies to quantify their support. For DDEV, this transparency can bridge the gap between reliance and sustainability.
+
+## Challenges with Current Models
+
+Tools like [thanks.dev](https://thanks.dev) and [Tidelift](https://tidelift.com/) attempt to address open-source funding by analyzing build files and distributing donations. While effective for libraries, these models fall short for tools like DDEV, which exist outside dependency management systems.
+
+DDEV isn’t listed in `package.json` or `composer.json` files; it’s an environment where development happens. This distinction means DDEV doesn’t benefit from algorithmic funding models, making direct acknowledgment and support from users critical.
+
+## A Call to Action
+
+For companies and agencies using DDEV, adopting the Open Source Pledge isn’t just about numbers -- it’s about recognizing the value of all the tools that enable your success. By taking the pledge, you’re contributing to the sustainability of an ecosystem that empowers your work.
+
+1. Sign the [Open Source Pledge](https://opensourcepledge.com/join/)
+2. Choose the portion that should be allocated to make DDEV a sustainable project and commit to its funding.
+
+We know that these choices require organizational action, and we're happy to help communication with your organization. DDEV's financial affairs are handled through the US nonprofit [DDEV Foundation](/foundation) so it's easy to fund, with clear management and reporting. We're happy to talk to your boss about any facet of this process, see the [Contact page](/contact).
+
+## Look to the Future!
+
+Imagine a day when the tools you rely on for development—-like DDEV—-suddenly (or gradually) stop working reliably. You've seen this happen before. Without sustainable funding, this could be a reality for countless open-source projects. We must all work together to support this critical ecosystem. You count on expert, up-to-date maintenance and improvement of the project to support your clients. It's time to figure out how to make that maintenance sustainable.
```

Comment:
```suggestion
Imagine a day when the tools you rely on for development--like DDEV--suddenly (or gradually) stop working reliably. You've seen this happen before. Without sustainable funding, this could be a reality for countless open-source projects. We must all work together to support this critical ecosystem. You count on expert, up-to-date maintenance and improvement of the project to support your clients. It's time to figure out how to make that maintenance sustainable.
```